ecl/key/FamilyTest.xml

```diff
@@ -0,0 +1,8 @@
+<Dataset name='TestName'>
+ <Row><TestName>GLM -- FamilyTest</TestName></Row>
+</Dataset>
+<Dataset name='Result'>
+ <Row><Result>Pass</Result></Row>
+</Dataset>
+<Dataset name='Errors'>
+</Dataset>
```

Comment:
Okay, at one '\n' (0x0A) char needed at the end of that line. I know Windows uses '\r\n' by default.